pkg/monitoring/metrics.go

```diff
@@ -18,3 +21,21 @@ func init() {
 func GetMetricsAddress() string {
 	return fmt.Sprintf("%s:%s", monitoringAddress, monitoringPort)
 }
+
+func GetMonitoringNamespace() string {
+	monitoringNamespaceFromEnv := os.Getenv("MONITORING_NAMESPACE")
+
+	if monitoringNamespaceFromEnv != "" {
+		return monitoringNamespaceFromEnv
+	}
+	return defaultMonitoringNamespace
+}
+
+func GetMonitoringServiceAccount() string {
+	monitoringServiceAccountFromEnv := os.Getenv("MONITORING_SERVICE_ACCOUNT")
+
+	if monitoringServiceAccountFromEnv != "" {
+		return monitoringServiceAccountFromEnv
+	}
+	return defaultServiceAccountName
+}
```

Comment:
We are already at "monitoring" package we can do `monitoring.GetNamespace` `monitorig.GetServiceAccount`

Reply:
Done